frontend/packages/console-app/src/components/terminal/redux/reducer.ts

```diff
@@ -0,0 +1,16 @@
+import { Map } from 'immutable';
+import { TerminalAction, Actions } from './action';
+
+export type State = Map<string, any>;
+
+export default (state: State, action: TerminalAction) => {
+  if (!state) {
+    return Map({
+      terminal: () => {},
+    });
+  }
+  if (action.type === Actions.TerminalDrawerToggleExpanded)
+    return state.setIn(['terminal', 'isExpanded'], !state.getIn(['terminal', 'isExpanded']));
```

Comment:
change this according to the above state.

Reply:
updated